src/views/ProductDetail.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
```

Comment:
I think we stopped doing this right?

Reply:
I think I missed removing it. I have done the suggested change sir.